docs/dm/zh_CN/features/relay-log.md

```diff
@@ -0,0 +1,111 @@
+relay-log
+===
+
+### 索引
+- [功能介绍](#功能介绍)
+- [本地存储目录结构](#本地存储目录结构)
+- [初始同步规则](#初始同步规则)
+- [数据清理](#数据清理)
+
+
+### 功能介绍
+
+DM-worker 启动后会自动同步上游的 binlog 到配置的本地目录（使用 `DM-Ansible` 部署的默认同步目录为 `<deploy_dir>/relay_log`），与 [MySQL Slave Relay Log](https://dev.mysql.com/doc/refman/5.7/en/slave-logs-relaylog.html) 的使用方式类似。DM-worker 运行过程中会实时同步上游的 binlog 更新到本地文件，syncer 组件会实时读取本地的 relay-log 更新并同步更新至下游数据库。
+
+
+### 本地存储目录结构
+
+以下是一个 relay-log 本地存储的目录结构示例：
+
+```
+<deploy_dir>/relay_log/
+|-- 7e427cc0-091c-11e9-9e45-72b7c59d52d7.000001
+|   |-- mysql-bin.000001
+|   |-- mysql-bin.000002
+|   |-- mysql-bin.000003
+|   |-- mysql-bin.000004
+|   `-- relay.meta
+|-- 842965eb-091c-11e9-9e45-9a3bff03fa39.000002
+|   |-- mysql-bin.000001
+|   `-- relay.meta
+`-- server-uuid.index
+```
+
+- subdir 目录：DM-worker 从上游数据库同步的 binlog 会保存在同一个目录，每一个目录叫做一个 subdir，subdir 的命名格式是 <上游数据库 uuid>.<本地 subdir 序号>。当上游进行[主从切换](./master-slave-switch.md)后，DM-worker 会生成一个新的 subdir 目录，并且新目录序号递增。例如在例子中，`7e427cc0-091c-11e9-9e45-72b7c59d52d7.000001` 目录里，`7e427cc0-091c-11e9-9e45-72b7c59d52d7` 表示上游数据库 uuid，000001是本地 subdir 序号。
```

Comment:
we have not added `master-slave-switch.md` right?

Reply:
yes, `feature` for `master-slave-switch` is not ready